docs/WRAPPER_REACT_QUERY.md

```diff
@@ -35,15 +35,20 @@ export const clientPersister = createSyncStoragePersister({ storage: clientStora
 3. Use created `clientPersister` in your root component (eg. `App.tsx`)
 
 ```ts
-import { PersistQueryClientProvider } from '@tanstack/react-query-persist-client'
+import { persistQueryClient } from "@tanstack/react-query-persist-client";
+
+persistQueryClient({
+  queryClient,
+  persister: clientPersister,
+});
 
 const App = () => {
   return (
-    <PersistQueryClientProvider persistOptions={{ persister: clientPersister }}>
+    <QueryClientProvider client={queryClient}>
```

Comment:
`QueryClientProvider` is imported from "@tanstack/react-query".

I didn't add it since it's required even for basic usage as well.

@mrousavy should I add this to the top?

```tsx
import { QueryClientProvider } from "@tanstack/react-query";
```

Reply:
Yes please add that - then we can merge :)